app/src/main/java/gov/hhs/cdc/trustedintermediary/external/jjwt/JjwtEngine.java

```diff
@@ -113,6 +114,29 @@ public void validateToken(String jwt, String encodedKey)
         }
     }
 
+    @Override
+    public boolean isValidAccessToken(String jwt, String privateKey) {
+
+        try {
+            Jws<Claims> claimsJws =
+                    Jwts.parserBuilder()
+                            .setSigningKey(readPrivateKey(privateKey))
+                            .build()
+                            .parseClaimsJws(jwt);
+
+            return LocalDateTime.now()
+                    .isBefore(
+                            claimsJws
```

Comment:
Another option here is to call the getExpirationDate() and passed the jwt param. But we might get a warning that we are not using the claimsJws object.